modules/customize-certificates-add-service-serving-crd.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="add-service-certificate-crd_{context}"]
+= Add the service CA bundle to a Custom Resource Definition
+
+You can annotate a Custom Resource Definition (CRD) with `service.beta.openshift.io/inject-cabundle=true` to have its `caBundle` field populated with the service CA bundle. This allows the Kubernetes API server to validate the service CA certificate used to secure the targeted endpoint.
+
+[NOTE]
+====
+The service CA bundle will only be injected into the CRD if the CRD is configured to use a webhook for conversion that is secured with a service CA certificate.
+====
+
+.Procedure
+
+. Annotate the CRD with `service.beta.openshift.io/inject-cabundle=true`:
++
+----
+$ oc annotate crd <crd-name> \//<1>
+     service.beta.openshift.io/inject-cabundle=true
+----
+<1> Replace `<crd-name>` with the name of the CRD to annotate.
++
+For example, use the following command to annotate the CRD `foo`:
++
+----
+$ oc annotate crd foo service.beta.openshift.io/inject-cabundle=true
+----
+
+. View the CRD to ensure that the certificate has been generated:
++
+----
+$ oc get crd <crd-name> -o yaml
+----
++
+It is displayed as `caBundle` in the YAML output:
```

Comment:
The CA bundle will appear in the `spec.conversion.webhook.clientConfig.caBundle` field in the YAML output.